src/ext.rs

```diff
@@ -50,22 +45,6 @@ macro_rules! if_else {
     };
 }
 
-// -----------------------------------------------------------------------------
-// Tokio
-// -----------------------------------------------------------------------------
-
-/// Creates a new Tokio runtime with the specified configuration.
-pub fn new_tokio_runtime(thread_name: &'static str, num_async_threads: usize, num_blocking_threads: usize) -> Runtime {
```

Comment:
Removed it from here because we should not create multiple Tokio runtimes, so the reuse does not make sense.